flink-staging/flink-ml/pom.xml

```diff
@@ -46,9 +46,9 @@
 		</dependency>
 
 		<dependency>
-			<groupId>com.github.fommil.netlib</groupId>
```

Comment:
Because Breeze depends on it and thus it is pulled transitivly.
On Mar 26, 2015 10:05 PM, "Henry Saputra" <notifications@github.com> wrote:

> In flink-staging/flink-ml/pom.xml
> <https://github.com/apache/flink/pull/539#discussion_r27256008>:
>
> > @@ -46,9 +46,9 @@
> >  		</dependency>
> >
> >  		<dependency>
> > -			<groupId>com.github.fommil.netlib</groupId>
>
> Just curious, why was this deleted?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/flink/pull/539/files#r27256008>.
>